vignettes/plotting_vignette.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Plotting Functionality in tcplfit2"
+author: "Center for Computational Toxicology and Exposure"
+output:
+   prettydoc::html_pretty:
+    theme: architect
+    toc: yes
+    toc_depth: 4
+vignette: >
+  %\VignetteIndexEntry{Plot_with_tcplfit2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, packages, warning=FALSE, message=FALSE}
+library(tcplfit2)
+library(dplyr)
+library(stringr)
+library(ggplot2)
+library(gridExtra)
+```
+
+```{r, include=FALSE}
+source("~/CompTox-ToxCast-tcplFit2/R/concRespPlot2.R")
+source("~/CompTox-ToxCast-tcplFit2/R/plot_allcurves.R")
+```
+
+
+# Introduction
+This vignette demonstrates how to plot concentration-response curves with two `ggplot2` based plotting functions in `tcplfit2`, namely `plot_allcurves` and `concRespPlot2`. 
```

Comment:
Suggested rewording: "curves with plotting functions in `tcplfit2`."
We may want to not limit this vignette just to the ggplot2 based functions in future.

Reply:
I think you may also want to mention that the vignette also provides demonstrations of how to customize plots in particular the ggplot2 based functions. Only needs to be a sentence.